readme.md

```diff
@@ -74,34 +97,131 @@ Slots allow you to add custom html into specific "slots" within the web componen
 | `read-along-header`     | The read along header | `span`              |
 | `read-along-subheader`  | Subheader (ie authors)| `span`              |
 
+## Page layout
+
+By default, the pages are two column layout with image on the left and text on the left. You force any page to one
+column layout by setting the class of the page to ```one-column-layout-page```
+
+```xml
+
+<div type="page" class="one-column-layout-page">
+  ...
+</div>
+```
+
+The default layout is auto adjust without restrictions. To force a 40-60 split between the image and text use
+the ```two-column-layout-page``` for the page.
```

Comment:
I'm confused by this wording; do I need to state:

```html
<div type="page" class="two-column-layout-page">
```

or

```
<div type="two-column-layout-page">
```

Please provide the full example, as I think that will make it clearer.

Reply:
added an example